app/src/pages/inside/dashboardPage/dashboardList/dashboardTable/dashboardTableColumns.jsx

```diff
@@ -86,6 +93,8 @@ export const DuplicateColumn = track()(
   injectIntl(({ value, customProps, className, tracking: { trackEvent }, intl }) => {
     const [opened, setOpened] = useState(false);
     const dropdownRef = useRef(null);
+    const activeProject = useSelector(activeProjectSelector);
+    const responseToCopy = useFetchedResponse(URLS.dashboardConfig(activeProject, value.id));
```

Comment:
I would suggest to make the call for config on opening popover, so the hook may return the async function to call when needed.
It may be called in `useEffect` with `opened` dependency and awaited in the `handleCopyConfig` callback.
That's will reduce the tons of unnecessary api calls for configs while just opening the dashboards page.